mrtt-ui/src/components/ProjectDetailsForm/ProjectDetailsForm.js

```diff
@@ -0,0 +1,202 @@
+// import { useEffect } from 'react'
+import styles from './style.module.scss'
+import { useForm, Controller } from 'react-hook-form'
+import { yupResolver } from '@hookform/resolvers/yup'
+import * as Yup from 'yup'
+import countries from '../../data/countries.json'
+import {
+  TextField,
+  Checkbox,
+  FormLabel,
+  InputLabel,
+  RadioGroup,
+  Radio,
+  FormControlLabel,
+  Button,
+  Stack
+} from '@mui/material'
+import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFns'
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider'
+import { MobileDatePicker } from '@mui/x-date-pickers/MobileDatePicker'
+import Autocomplete from '@mui/material/Autocomplete'
+
+function ProjectDetailsForm() {
+  // form validation rules
+  const validationSchema = Yup.object().shape({
+    projectTitle: Yup.string().required('Project title is required'),
+    projectAims: Yup.array().of(Yup.string()).typeError('Select at least one item'),
+    hasProjectEndDate: Yup.boolean(),
+    projectStartDate: Yup.string().required('Select a start date'),
+    projectEndDate: Yup.string().when('hasProjectEndDate', {
+      is: true,
+      then: Yup.string().required('Please select an end date')
+    }),
+    countries: Yup.array()
+      .of(
+        Yup.object().shape({
+          name: Yup.string(),
+          code: Yup.string()
+        })
+      )
+      .min(1)
+      .typeError('Select at least one item')
```

Comment:
Can you be specific with what needs to be selected?
```suggestion
      .typeError('Select at least one country')
```